// api/api.proto

syntax = "proto2";
package codingtag;

// services -> hardware (MessageType 1):
message SendIR {
	required uint32 dummy = 1;
}

// hardware -> services (MessageType 2):
message ReceiveIR {
	required uint32 src_client_id = 1;
}
